Packs/CommonWidgets/Widgets/widget-LargestInputsAndOuputsInIncidentsWidget.json

```diff
@@ -0,0 +1,59 @@
+{
+ "id": "LargestInputsAndOuputsInIncidentsWidget",
+ "version": -1,
+ "modified": "2020-08-18T13:31:21.939387+03:00",
+ "name": "Largest Inputs And Ouputs In Incidents",
+ "dataType": "scripts",
+ "widgetType": "table",
+ "query": "GetLargestInputsAndOuputsInIncidents",
+ "dateRange": {
+  "fromDate": "0001-01-01T00:00:00Z",
```

Comment:
should this be filled?

Reply:
It was set up like that via `sdk format` and it seems other widgets are built the same way